### PR TITLE
Cone detection yolov8

### PR DESCRIPTION
**Pull request by ugsfume**

### CODE REVIEW

File: ros/cone_detection/CODE_OF_CONDUCT.md
Comment (daniel-bara) [inline ros/cone_detection/CODE_OF_CONDUCT.md:1]: Do we need this file?

File: ros/cone_detection/launch/kitti_tracker.launch
Comment (daniel-bara) [inline ros/cone_detection/launch/kitti_tracker.launch:1]: Let's delete these kitti_* launch files if the only difference is the model

File: ros/cone_detection/launch/kitti_tracker_with_cloud.launch
Comment (daniel-bara) [inline ros/cone_detection/launch/kitti_tracker_with_cloud.launch:1]: Let's delete these kitti_* launch files if the only difference is the model

File: ros/cone_detection/models/conev8s.pt
Comment (daniel-bara) [inline ros/cone_detection/models/conev8s.pt:1]: Do we expect to use this model? Afaik the n version works just as well

File: ros/cone_detection/src/tracker_with_cloud_node.cpp
Comment (daniel-bara) [inline ros/cone_detection/src/tracker_with_cloud_node.cpp:1]: Can we name this `tracker_with_point_cloud`? I always think of cloud computing when I see `tracker_with_cloud`

File: ros/cone_detection/CODE_OF_CONDUCT.md
Comment (ugsfume) [reply ros/cone_detection/CODE_OF_CONDUCT.md:1]: nope, removed

File: ros/cone_detection/models/conev8s.pt
Comment (ugsfume) [reply ros/cone_detection/models/conev8s.pt:1]: I want to keep it here for now to compare their performance in outdoor and high-speed settings.

File: ros/cone_detection/src/tracker_with_cloud_node.cpp
Comment (ugsfume) [reply ros/cone_detection/src/tracker_with_cloud_node.cpp:1]: Longer launch file names seem inefficient. Deal with it

**Review by daniel-bara — APPROVED** [review]

**Pull request merged**